manifests/g/GorrilaDevs/Ferium/4.5.0/GorrilaDevs.Ferium.installer.yaml

```diff
@@ -0,0 +1,20 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: GorrilaDevs.Ferium
+PackageVersion: 4.5.0
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: ferium.exe
+Dependencies:
+  PackageDependencies:
+    - PackageIdentifier: Microsoft.VCRedist.2015+.x64

```

Comment:
Is the VS runtime actually required? If so, could I know how to determine the runtime dependencies of an executable?

Reply:
For me this works without that, but I might have a different version for it from one of the games I have installed.